source/openapi3/split.yaml

```diff
@@ -4428,6 +4436,13 @@ components:
           type: boolean
           description: Enforce prechecking of available funds before approving the Payment Request. see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)
           example: "false"
+        your_bank_account_id:
+          type: string
+          description: Refers to `initiator_bank_account_id`. You can use this to identify issues related to the initiator's bank account.
```

Comment:
I find this description confusing. I'm thinking something along the lines:

Specify bank account id where to receive the funds if you want to use non Primary account.

ditto others

Reply:
Just another option since we can use some floats for payment requests and payments

Specify which bank account we should use to settle/disburse the funds for this transaction. If blank, your primary account will be used.

Reply:
I used @astley92 's description since it is the most explicit one :)